app/controllers/application_controller.rb

```diff
@@ -1,2 +1,6 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    @current_user ||= User.first
+  end
+  helper_method :current_user
```

Comment:
- [ ] I think we don't need to condition it here, all we need is the first user, just a demo. So I will encourage you to just have it as 
```
def current_user
  User.first
end
```

Reply:
class ApplicationController < ActionController::Base
  def current_user
    User.first
  end
  helper_method :current_user
end